.circleci/config.yml

```diff
@@ -128,7 +128,7 @@ jobs:
           name: Run sample tests.
           command: npm run samples-test
           environment:
-            GCLOUD_PROJECT: node-gcloud-ci
+            GCLOUD_PROJECT: long-door-651
```

Comment:
@kinwa91 should this be getting changed?  Synth bug?

Reply:
Good catch! The `test_project` arg never got shipped: 
https://github.com/GoogleCloudPlatform/synthtool/pull/63